core/src/data/dataSource.cpp

```diff
@@ -7,7 +7,7 @@
 #include "tile/tileManager.h"
 #include "labels/labels.h"
 
-//---- DataSource Implementation----
+namespace Tangram {
```

Comment:
Also I was reading more on this ... Its recommended not to use "using" unless really needed and that too in limited scopes. So I am good with this.